README.md

```diff
@@ -223,7 +223,7 @@ The following inputs can be used as `step.with` keys
 | Name             | Type    | Description                        |
 |------------------|---------|------------------------------------|
 | `app_port` | String | Port to be expose for the container. Default is `3000` | 
-| `app_directory` | String | Relative path for the directory of the app. (i.e. where the `docker-compose.yaml` file is located). This is the directory that is copied into the EC2 instance. Default is `/`, the root of the repository. |
+| `app_directory` | String | Relative path for the directory of the app. (i.e. where the `docker-compose.yaml` file is located). This is the directory that is copied into the EC2 instance. Default is `/`, the root of the repository. Add a `.ignore` file with a list of files to be exluded. |
```

Comment:
As an actually technical comment, from what i gather on the webs, the `--exclude-from` config uses the typical glob patterns. It might be useful to include that info here. Also, that we're specifically looking for a `.ignore` file; `rsync` can use any text file as input. If we're going to hardcode, we might want something more specific like `.bitops-ignore`.